app/src/views/table/RestoreTables.vue

```diff
@@ -369,8 +369,8 @@ const progress = reactive({
 });
 
 const credentials = reactive({
-  AWS_REGION: "",
-  AWS_ENDPOINT: "",
+  AWS_REGION: "us-west-2",
+  AWS_ENDPOINT: "https://dynamodb.us-west-2.amazonaws.com",
```

Comment:
This could result in user understanding that we can only restore from AWS, and AWS has other URLs and regions to restore from [here](https://docs.aws.amazon.com/general/latest/gr/ddb.html).

We could also restore from some other URLs, for example a local system, I recently ended up restoring from http://192.168.1.26:8000.

I suggest not to hard-code any value and let it stay as an example. An improvement would be, a free text box with a dropdown to select pre-defined urls from AWS and also allow users to type in custom URLs.

![image](https://github.com/kritish-dhaubanjar/dynamodb-dashboard/assets/25634165/413161e9-d92b-456d-aed8-75de63efead1)

Thanks!

Reply:
Something close to 

![image](https://github.com/kritish-dhaubanjar/dynamodb-dashboard/assets/25634165/a0d25761-418a-439b-bc0d-376068b276d5)